doc/source/devel/make_release.rst

```diff
@@ -130,11 +130,9 @@ Release checklist
 * Check on different platforms, particularly windows and PPC. Look at the
   `nipy buildbot`_ automated test runs for this.
 
-* Check the documentation doctests::
+* Check the documentation doctests (forcing Python 2)::
```

Comment:
Once we get the other one in, I'll remove this change from this PR. No particular reason to add this to the history if it'll only last until the other PR goes in.